test/queue_tests.py

```diff
@@ -10,17 +10,17 @@
 
 class WhenDeclaringAQueue(OpenChannelContext):
     def when_I_declare_a_queue(self):
-        self.async_partial(self.channel.declare_queue('my.nice.queue', durable=True, exclusive=True, auto_delete=True))
+        self.async_partial(self.channel.declare_queue('my.nice.queue', durable=True, exclusive=True, auto_delete=True, arguments={'x-expires': 300, 'x-message-ttl': 1000}))
 
     def it_should_send_a_QueueDeclare_method(self):
-        expected_method = spec.QueueDeclare(0, 'my.nice.queue', False, True, True, True, False, {})
+        expected_method = spec.QueueDeclare(0, 'my.nice.queue', False, True, True, True, False, {'x-expires': 300, 'x-message-ttl': 1000})
```

Comment:
Certainly, that would be good. I'll see what I can do. I fear a bit getting the timing right, so the test doesn't become too long, as I have to wait for messages/queues to expire.